rdt-open-reader/src/test/java/io/ona/rdt/robolectric/activity/PatientRegisterActivityTest.java

```diff
@@ -74,6 +76,7 @@ public void testOnRequestPermissionsResultShouldAlertPermissionsAreRequiredIfDen
 
     @Test
     public void testOnCreateShouldCorrectlyInitializeActivity() {
+        LangUtils.saveLanguage(patientRegisterActivity, BuildConfig.LOCALE);
```

Comment:
This is repeated for all Activity tests, you can do this instead https://github.com/onaio/rdt-standard/pull/579/files#r509921665.